community-connectors/google-fit/Connector.js

```diff
@@ -0,0 +1,572 @@
+// vim: ft=javascript:ts=2:sw=2
+/*
+Copyright 2017 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+var OAUTH_CLIENT_ID = "<oauth client id>";
```

Comment:
Prefer using Script Properties for storing client_id and client_secret. To do this:

1. Visit [Apps Script](https://script.google.com), then click on **File** -> **Project
   Properties** -> **Script Properties**.
2. Add the following key-value pairs.

  | Key                   | Value                |
  |-----------------------|----------------------|
  | `OAUTH_CLIENT_ID`     | {YOUR CLIENT ID}     |
  | `OAUTH_CLIENT_SECRET` | {YOUR CLIENT SECRET} |

You can then access these fields from Apps Script as follows:
```javascript
var scriptProps = PropertiesService.getScriptProperties();
var clientId = scriptProps.getProperty('OAUTH_CLIENT_ID');
var clientSecret = scriptProps.getProperty('OAUTH_CLIENT_SECRET');
```

Reply:
Done